src/functionalTest/resources/request/casedata/ccd-callback-casedata-caseworker-submit-case.json

```diff
@@ -0,0 +1,72 @@
+{
+  "data": {
+    "cicCaseCaseCategory": "Assessment",
+    "cicCaseCaseSubcategory": "other",
+    "cicCaseCaseReceivedDate": "2000-01-01",
+    "cicCasePartiesCIC": [
+      "ApplicantCIC",
+      "SubjectCIC",
+      "RepresentativeCIC"
+    ],
+    "cicCaseFullName": "Testing subject",
+    "cicCasePhoneNumber": "01234567891",
+    "cicCaseDateOfBirth": "2000-01-01",
+    "cicCaseAddress": {
+      "AddressLine1": "Buckingham Palace",
+      "AddressLine2": "",
```

Comment:
Seeing as you are using IGNORING_EXTRA_FIELDS you can get rid of all these empty strings in this json file